docs/src/tutorial/custom-data.rst

```diff
@@ -0,0 +1,162 @@
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
+
+We will retrieve 10 sequences from GISAID's EpiCoV database.
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__ and select **Login**.
+
+   .. image:: ../images/gisaid-homepage.png
+      :width: 400
+      :alt: GISAID login link
+
+2. Login to your GISAID account.
+
+   .. image:: ../images/gisaid-login.png
+      :width: 200
+      :alt: GISAID login
+
+3. In the top left navigation bar, select **EpiCoV** then **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+4. Select the first 10 records.
+
+   .. note::
+
+      You can filter and select a different set of records here, but note that more records will take the workflow longer to run, and the GISAID web interface has a maximum of 5,000 records per download.
+
+   .. image:: ../images/gisaid-select-records.png
+      :width: 700
+      :alt: GISAID EpiCoV Search
+
+5. Select **Download** in the bottom right of the search results.
+6. Select **Input for the Augur pipeline** as the download format.
+7. Select **Download**.
+
+   .. image:: ../images/gisaid-augur-pipeline-download.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+   .. note::
+
+      You may see different download options, but it is fine as long as **Input for the Augur pipeline** is available.
+
+8. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
+9.  Move these extracted files to the ``data/`` directory in the ``ncov/`` directory.
+10. Rename the files as ``custom.metadata.tsv`` and ``custom.sequences.fasta``.
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a pre-written ``--configfile``:
+
+.. code:: text
+
+   nextstrain build . --cores 4 --configfile ncov-tutorial/custom-data.yaml
+
+Break down the command
+~~~~~~~~~~~~~~~~~~~~~~
+
+The workflow can take several minutes to run. While it is running, you can investigate the contents of ``custom-data.yaml`` (comments excluded):
+
+.. code-block:: yaml
+
+   inputs:
+     - name: reference_data
+       metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+       sequences: https://data.nextstrain.org/files/ncov/open/reference/sequences.fasta.xz
+     - name: custom_data
+       metadata: data/custom.metadata.tsv
+       sequences: data/custom.sequences.fasta
+
+   refine:
+     root: "Wuhan-Hu-1/2019"
+
+   builds:
+     custom-build:
+       title: "Build with custom data and example data"
+       auspice_config: ncov-tutorial/auspice-config-custom-data.json
```

Comment:
I referenced the auspice config JSON file as a path relative to `ncov/` so it is prefixed with `ncov-tutorial/`. This works fine as long as readers follow the directions in the previous tutorial, but it's a bit weird to think about when looking at the contents of `ncov-tutorial/` directly without context that the workflow is being run one directory above.

Reply:
Placing `ncov/` within `ncov-tutorial/` could make this more straightforward, but this is a new approach which needs some tinkering to get it working (see [Slack thread](https://bedfordlab.slack.com/archives/C02Q4Q5S85A/p1646334194519489?thread_ts=1646334170.382399&cid=C02Q4Q5S85A)).

Reply:
Agreed that it is a little weird without context, although the way it's written here is great until we figure out another repo/configs organization.

If the `ncov-tutorial/` directory were instead named `my-builds/` or `my-analyses/`, the resulting path is a little less strange to me (as in, "these are my custom files to control the workflow" instead of "the tutorial lives inside of the workflow, for some reason").

```yaml
builds:
   custom-build:
     title: "Build with custom data and example data"
     auspice_config: my-analyses/auspice-config-custom-data.json
```

This kind of naming convention is part of what I was thinking with treating the `ncov-tutorial` repo like a template, so folks can reference their own set of files as they like.

Reply:
Done: 7e68a150

Reply:
Relatedly, do you think we should instruct users to create their own repo from of the `ncov-tutorial` template repo in the first example or at the end?

I think the end would be good enough since we aren't instructing any file changes in these tutorials.

Reply:
The experience here has been updated based on https://github.com/nextstrain/ncov/pull/894#discussion_r853401870